libs/hdf-converters/data/converters/cciListXml2json.ts

```diff
@@ -0,0 +1,141 @@
+import fs from 'fs';
+import * as _ from 'lodash';
+import xml2js from 'xml2js';
```

Comment:
This happymapper class is the structure of the cci xml - https://github.com/mitre/inspec_tools/blob/master/lib/happy_mapper_tools/cci_attributes.rb - may be a useful reference. 

There is also xpath function and other useful functions which may be useful if converted to ts? Maybe.

Reply:
https://github.com/mitre/inspec_tools/blob/master/lib/happy_mapper_tools/cci_attributes.rb

Reply:
might be OBE since lines 112-115 also does a maxBy on the version of the CCI reference